RichClipboard.package/RCClippingsWindow.class/instance/toggleDateSort.st

```diff
@@ -2,10 +2,14 @@ sort-date
 toggleDateSort
 	self dateSortInverted ifTrue: [ 
 		dateSortInverted := false.
-		self clippingsList sortByDate
+		self clippingsList sortByDate. 
+		self currentSortMode: ' (new-old)'.
+		self updateWindowTitle.
 	] ifFalse: [
 		dateSortInverted := true.
-		self clippingsList sortByDateInverse
+		self clippingsList sortByDateInverse. 
+		self currentSortMode: ' (old-new)'.
+		self updateWindowTitle.
 	].
 	self changed: #dateSortButtonLabel.
 	self fixHeightOfToolbarButtons
```

Comment:
Das "self updateWindowTitle" kann unten aufgerufen werden (und muss nicht in beiden Fallunterscheidungen einzeln hingeschrieben werden)

Reply:
Nachher die hinteren Punkte in den Klammern entfernen